src/components/MonsterDetails.js

```diff
@@ -1,27 +1,33 @@
 import React from 'react'
-import styled from 'styled-components'
+import MonsterDetailsHead from './MonsterDetailsHead'
 import MonsterDetailsStat from './MonsterDetailsStats'
+import Proficiencies from './Proficiencies'
+import ActionsAbilities from './ActionsAbilities'
 
 export default function MonsterDetails({ monster }) {
   return (
     <>
-      <Headline>{monster.name}</Headline>
+      <MonsterDetailsHead
+        name={monster.name}
+        type={monster.type}
+        cr={monster.challenge_rating}
```

Comment:
`cr` is a little hard to understand (at least for me). Maybe you can change it to `challengeRating`
and I guess you should use camelCase for `challenge_rating'

```suggestion
        challengeRating={monster.challengeRating}
```

Reply:
You are not the first to mention that. Maybe i should actually do that even if its displayed as CR in the App and the game.

"challenge__rating" is the name of a property in my Object i get from the API. I cant change that or it would return undefined.